GettyImages.Api/SdkException.cs

```diff
@@ -50,15 +47,6 @@ internal static async Task GenerateSdkExceptionAsync(HttpResponseMessage httpRes
             }
             switch (httpResponse.StatusCode)
             {
-                case HttpStatusCode.Forbidden:
```

Comment:
I'm assuming this was just some corner case in Mashery where calls (some? all?) that violated the QPS throttle were tossed back as a `401` with some Mashery-specific header.

Reply:
That is a correct assumption. They didn't use the `429` response code for some reason.